src/tests/tracing/eventlistener/EventListenerThreadPool.cs

```diff
@@ -40,6 +41,10 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
             {
                 Interlocked.Increment(ref TPWorkerThreadWaitCount);
                 TPWaitEvent.Set();
+            } else if (eventData.EventName.Equals("IOPack"))
```

Comment:
I get an assert [here ](https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/Runtime/EHHelpers.cpp#L578)when I run this test.